tokens/0xc5bbae50781be1669306b9e001eff57a2957b09d.yaml

```diff
@@ -0,0 +1,19 @@
+---
+addr: '0xc5bbae50781be1669306b9e001eff57a2957b09d'
+decimals: 5
+description: >-
+  The Decentralized Universal Gifting Protocol for Global Content Creators.
+links:
+- Blog: https://medium.com/@GIFTO
+- CoinMarketCap: https://coinmarketcap.com/currencies/gifto/
+- Email: mailto:info@gifto.io
+- Facebook: https://www.facebook.com/gifto.io/
+- Github: https://github.com/GIFTO-io
+- Reddit: https://www.reddit.com/user/GIFTO-io/
+- Telegram: https://t.me/GIFTOOfficial
+- Twitter: https://twitter.com/GIFTO_io
+- Website: https://gifto.io/
```

Comment:
Misconfigured HTTPS causing failure, token team notified. I'd try to pull this token out into a separate PR so we get tests passing on most tokens and a specific reason for one token merged with failing tests.

Reply:
Done: https://github.com/forkdelta/tokenbase/pull/565